cmd/tempo/app/modules.go

```diff
@@ -229,13 +227,6 @@ func (t *App) initMemberlistKV() (services.Service, error) {
 		ring.GetCodec(),
 	}
 
-	hostname, err := os.Hostname()
```

Comment:
Double-checking :)

The package does this by default too? Seeing the commit history, I suppose you tried it!

Reply:
yup, i removed it, added it back and removed it again.

the package does indeed already use hostname by default. good :eyes: !